### ISSUE TITLE
The module ""vue"" has no exported member "VueConstructor".

### ISSUE THREAD

**Issue by 1622583480** [SortableJS/vue.draggable.next#75]
my env 
eletron + vite + typescript +vue3
This is just an empty project
The following translation is from machine translation, and may be somewhat inaccurate
![image](https://user-images.githubusercontent.com/59474029/129313427-ad2528bf-669f-4e76-bce9-d0621c3980c4.png)

In an empty vue file in vue3
```vue
<template></template>
<script setup lang="ts">
import draggable from "vuedraggable"
</script>
```
Then when I run the package
node_modules/vuedraggable/src/vuedraggable.d.ts:3:37 - error TS2305: Module '"vue"' has no exported member 'VueConstructor'.

3   import { ComponentPublicInstance, VueConstructor } from 'vue';
![image](https://user-images.githubusercontent.com/59474029/129313271-4ff91854-d304-4329-bc9a-57057c88e96d.png)

**Comment by DehaiWang** [comment]
i meet the same question,do you solve the question?

**Comment by 1622583480** [comment]
> i meet the same question,do you solve the question?
No, because of this problem, I temporarily give up this plug-in🥲🥲🥲🥲
But this plugin is very easy to use

**Comment by David-Desmaisons** [comment]
Corrected in version 4.1.0
